core/src/main/java/org/bitcoinj/core/Transaction.java

```diff
@@ -1089,10 +1102,12 @@ public Sha256Hash hashForSignature(int inputIndex, byte[] redeemScript,
      * @param type Should be SigHash.ALL
      * @param anyoneCanPay should be false.
      */
-    public Sha256Hash hashForSignature(int inputIndex, Script redeemScript,
-                                                    SigHash type, boolean anyoneCanPay) {
-        int sigHash = TransactionSignature.calcSigHashValue(type, anyoneCanPay);
-        return hashForSignature(inputIndex, redeemScript.getProgram(), (byte) sigHash);
+    public Sha256Hash hashForSignature(
```

Comment:
Please don't reformat this method signature (and the 5 above).

Reply:
Added the reformat makes it easy to compare to segwit branch, Reversing.